src/ln/channelmanager.rs

```diff
@@ -341,6 +341,15 @@ pub struct ChannelManager {
 	logger: Arc<Logger>,
 }
 
+/// The amount of time we require our counterparty wait to claim their money (ie time between when
+/// we, or our watchtower, must check for them having broadcast a theft transaction).
+#[cfg(not(test))]
+pub(crate) const BREAKDOWN_TIMEOUT: u16 = 6 * 24;
```

Comment:
Hmm, cant you drop this duplication now? No way we need something to access something from outside the crate in a test build?